server-api/package.json

```diff
@@ -31,5 +32,10 @@
     "eslint-plugin-import": "2.17.2",
     "eslint-plugin-jsx-a11y": "6.2.1",
     "prettier-eslint-cli": "^4.7.1"
```

Comment:
Maybe add a linter script like
```
"lint-staged": {
    "linters": {
      "{src,scripts}/**/*.{js,jsx,json,scss,sass}": [
        "prettier --single-quote --write",
        "git add"
      ]
    }
  }
```

And then call it in the pre-commit
```
"husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  }
```

Reply:
@abhishek71994 done!